src/pyspextool/plot/plot_spectra.py

```diff
@@ -5,13 +5,13 @@
 from pyspextool.fit.robust_savgol import robust_savgol
 from pyspextool.plot.limits import get_spec_range
 from pyspextool.io.check import check_parameter, check_file
-from read_spectra_fits import read_spectra_fits
+from pyspextool.io.read_spectra_fits import read_spectra_fits
 from pyspextool.plot import limits
 
 def plot_spectra(file, plot_type='continuous', figsize=None, y='flux',
-                 aperture=0, xlabel=None, ylabel=None, title=None,
+                 aperture=0, xlabel='Wavelength', ylabel='Flux Density', title=None,
```

Comment:
There will be times when the spectrum will have units of pixels, and the raw data is DN s-1, which was why I left then as None.  Thoughts?

Reply:
since default y = flux, default ylabel = Flux density made sense; for x-axis, the very first spectrum that comes out of pyspextool already has wavelength as an axis, so None doesn't sound like the obvious default